snap/snapcraft.yaml

```diff
@@ -33,42 +35,28 @@ apps:
       CONFIG_PRO_ARG: "-cp=consul.http://localhost:8500"
       CONF_ARG: "--confdir=$SNAP_DATA/config/app-rfid-llrp-inventory/res"
       REGISTRY_ARG: "--registry"
-    plugs: [network, network-bind]
 
-parts:
-  go:
-    plugin: nil
-    source: snap/local
-    build-packages: [curl]
-    override-build: |
-      # use dpkg architecture to figure out our target arch
-      # note - we specifically don't use arch
-      case "$(dpkg --print-architecture)" in
-        amd64)
-          FILE_NAME=go1.15.2.linux-amd64.tar.gz
-          FILE_HASH=b49fda1ca29a1946d6bb2a5a6982cf07ccd2aba849289508ee0f9918f6bb4552
-          ;;
-        arm64)
-          FILE_NAME=go1.15.2.linux-arm64.tar.gz
-          FILE_HASH=c8ec460cc82d61604b048f9439c06bd591722efce5cd48f49e19b5f6226bd36d
-          ;;
-      esac
-      # download the archive, failing on ssl cert problems
-      curl https://dl.google.com/go/$FILE_NAME -O
-      echo "$FILE_HASH $FILE_NAME" > sha256
-      sha256sum -c sha256 | grep OK
-      tar -C $SNAPCRAFT_STAGE -xf go*.tar.gz --strip-components=1
-    prime:
-      - "-*"
+      SECRETSTORE_TOKENFILE:  /tmp/app-rfid-llrp-inventory/secrets-token.json
```

Comment:
Looks like some lines have been removed, possibly as a result of merge conflict resolution errors.

Line 39 should be replaced with:
```
      SECRETSTORE_TOKENFILE:  $SNAP_DATA/app-rfid-llrp-inventory/secrets-token.json
    passthrough:
      install-mode: disable
    plugs: [network, network-bind]
```

Reply:
I have re-added those lines